TribalWars/src/ApplicationCore/Helper/StringResources.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.Localization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Helper
```

Comment:
Other namespaces in ApplicationCore are named by plural form, but Helper is in singular form, that is a little bit inconsistent.